docs/index.md

```diff
@@ -1,5 +1,89 @@
-Steps to set up an analysis pipeline to analyze base editor validation experiments using [CRISPResso2](https://github.com/pinellolab/CRISPResso2) 
-are enumerated below:
+Steps to set up an analysis pipeline to analyze base editor validation experiments to run [CRISPResso2](https://github.com/pinellolab/CRISPResso2): <br/>
```

Comment:
Steps to set up an analysis pipeline to analyze base editor validation experiments using [CRISPResso2]